sources/lib/run-time/c-run-time.c

```diff
@@ -632,35 +583,11 @@ INLINE int function_number_required(dylan_simple_method* function) {
   return(signature_number_required(function->signature));
 }
 
-INLINE int function_number_values(dylan_simple_method* function) {
-  return(signature_number_values(function->signature));
-}
-
-INLINE int function_key_p(dylan_simple_method* function) {
-  return(signature_key_p(function->signature));
-}
-
-INLINE int function_all_keys_p(dylan_simple_method* function) {
-  return(signature_all_keys_p(function->signature));
-}
-
-INLINE int function_rest_p(dylan_simple_method* function) {
-  return(signature_rest_p(function->signature));
-}
-
 INLINE int function_optionals_p(dylan_simple_method* function) {
   return(signature_optionals_p(function->signature));
 }
 
 
-INLINE int function_rest_value_p(dylan_simple_method* function) {
-  return(signature_rest_value_p(function->signature));
-}
-
-INLINE int function_next_p(dylan_simple_method* function) {
-  return(signature_next_p(function->signature));
-}
-
```

Comment:
These and many of the other unused functions were useful under the debugger or for docs in how to access that data.

Reply:
Good point. I can add a comment to that effect and remove the INLINE which will stop them being flagged as unused. Does that sound OK?